crates/db_schema/src/source/tagline.rs

```diff
@@ -32,8 +32,15 @@ pub struct Tagline {
 #[derive(Clone, Default)]
 #[cfg_attr(feature = "full", derive(Insertable, AsChangeset))]
 #[cfg_attr(feature = "full", diesel(table_name = tagline))]
-pub struct TaglineForm {
+pub struct TaglineInsertForm {
   pub local_site_id: LocalSiteId,
   pub content: String,
-  pub updated: Option<DateTime<Utc>>,
+}
+
+#[derive(Clone, Default)]
+#[cfg_attr(feature = "full", derive(Insertable, AsChangeset))]
+#[cfg_attr(feature = "full", diesel(table_name = tagline))]
+pub struct TaglineUpdateForm {
+  pub content: String,
+  pub updated: Option<Option<DateTime<Utc>>>,
```

Comment:
This is because of a [comment](https://github.com/LemmyNet/lemmy/pull/4580/files/f1b993f90ed262b20e03bde4ed4233afd11695d0#r1553942482) by @dessalines that the column is a nullable.

Reply:
Correct, but also it should be `content: Option<String>`

To clarify:


_ | _
--- | ---
Inserts to a NOT NULL column | `String`
Inserts to a Nullable column | `Option<String>`
Updates to a NOT NULL column | `Option<String>`
Updates to a Nullable column | `Option<Option<String>`